include/net/nrf_cloud.h

```diff
@@ -1147,11 +1151,14 @@ int nrf_cloud_credentials_check(struct nrf_cloud_credentials_status *const cs);
 /**
  * @brief Check if the credentials required for connecting to nRF Cloud exist.
  *        The application's configuration is used to determine which credentials
- *        are required.
+ *        are required. Check the size of the root CA certificates installed
+ *        and deny connection if the required root CA certificate(s) are not
```

Comment:
```suggestion
 *        and return an error code if the size of the root CA certificate(s) is not appropriate for the configured transport type.
```

or something similar, since this function doesn't directly deny the connection.